tools/protodoc/protodoc.py

```diff
@@ -432,27 +432,27 @@ def strip_leading_space(s):
 
 def file_cross_ref_label(msg_name):
     """File cross reference label."""
-    return 'envoy_api_file_%s' % msg_name
+    return 'envoy_v3_api_file_%s' % msg_name
```

Comment:
this is basically your comment above i think - ie doing it in protodoc

Reply:
well, at least one interpretation of doing so these need to match the links essentially